config/co2eq_parameters_lifecycle.json

```diff
@@ -6576,6 +6576,13 @@
             "value": 199.07165644246462
           }
         ]
+      },
+	  "TH": {
+        "unknown": {
+          "_url": "https://www.iea.org/fuels-and-technologies/electricity",
+          "source": "IEA 2020; assumes 9.1% biomass, 19.3% coal, 64.7% gas, 2.5% hydro, 2.6% solar, 1.7% wind, 0.1% other",
+          "value": 498.9322479978950
```

Comment:
I don't get the same exact value:

https://docs.google.com/spreadsheets/d/16j5BVc4L34BDqVE0GeDBv0P3zQupT6zU77Kj5UK7GjE/edit?usp=sharing

Can you additionally please add the factor for direct co2eq as well and link both to the spreadsheet above as the source please? thanks

Reply:
For context, the full breakdown of the number I used for my CO2 lifecycle calculation is in `co2eq_parameter_all.json`
```json
 "TH": {
        "_source": "IEA 2020 (used because Our World in Data has a lot of 'other renewables' generation not broken down by type). Biomass includes waste.",
        "_url": "https://www.iea.org/data-and-statistics/data-browser?country=THAILAND&fuel=Energy%20supply&indicator=ElecGenByFuel",
        "powerOriginRatios": {
          "value": {
            "biomass": 0.0912401506077485,
            "coal": 0.1934589830217160,
            "gas": 0.6467002186068250,
            "geothermal": 0.0000053711750520,
            "hydro": 0.0248631693155512,
            "oil": 0.0007036239318076,
            "solar": 0.0264691506560890,
            "wind": 0.0165593326852115
          }
        }
      },
```
Whereas the 0.1% `other` are actually `geothermal` and `oil` here

Reply:
NVM, seems like my calculation is indeed wrong. I will update the number accordingly in this case. Thanks for the double check :)